src/services/sheet.service.ts

```diff
@@ -0,0 +1,32 @@
+import { fetchJson } from '../utils/fetch.ts';
+
+interface SheetsResponse {
```

Comment:
Hier musst du mal mit Samu sprechen - er hat ja jetzt eine "shared.types.ts" Datei erstellt in der er den SitemapResponse erstellt hat.

Reply:
ist auf develop gemerged. Du kannst develop reinmergen...